framework/src/modules/interoperability/utils.ts

```diff
@@ -156,6 +157,13 @@ export const rawStateStoreKey = (storePrefix: number) => {
 	return Buffer.concat([DB_KEY_STATE_STORE, moduleIDBuffer, storePrefixBuffer]);
 };
 
+export const isInboxUpdateEmpty = (inboxUpdate: InboxUpdate) =>
```

Comment:
shouldn't this be `&&` condition?

Reply:
I thought if any one of them is true then we can say that the inboxUpdate is empty

Reply:
So I have changed the condition to only flag inboxUpdate empty if all the properties are empty